base/server/man/man8/pki-server-nuxwdog.8

```diff
@@ -37,10 +37,10 @@ to remove this file entirely, and have the server prompt for these passwords on
 startup.  This means of course that it will not be possible to start the CS
 instance unattended, including on server reboots.
 .PP
-\fBnuxwdog\fR is a daemon that will launch the CS instance and prompt the administrator
-for the relevant passwords.  These passwords will be cached securely in the kernel
-keyring.  If the CS instance crashes unexpectedly, \fBnuxwdog\fR will attempt to restart
-the instance using the cached passwords.
+\fBnuxwdog\fR is a script that is launched before the CS instance starts and prompts
```

Comment:
The new 'nuxwdog' is not a daemon nor a script. It is a mechanism to start PKI server without storing passwords in file (i.e. password.conf).